local_server/common/aws_secret_utils.py

```diff
@@ -0,0 +1,23 @@
+import logging
```

Comment:
Why does the code base need the AWS secret manager?  (same question for the config code that uses it - eg, `aws_secrets_manager` config in `common/config/external_config.py`)?

Reply:
We wanted to continue supporting S3 access from the local server. I thought there could be use cases where you set some secrets via secretsmanager and didn't want to break that. Happy to take it out if that seems far-fetched though.